fastlane/Fastfile

```diff
@@ -361,13 +361,26 @@ REPOSITORY_NAME="WordPress-Android"
   #####################################################################################
   desc "Updates store metadata and runs the release checks"
   lane :finalize_release do | options |
-    app = options[:app]
+    app = options[:app] || 'wordpress'
```

Comment:
I've seen this is done in other lanes, so I added it for consistency

Reply:
👍  This will be refactored soon (once I get time to undo/redo the old approach that was added around `app:` parameter and instead make the various lanes always act on both WP and JP at once), but it's good to have in the meantime.